public/js/script.js

```diff
@@ -49,6 +49,22 @@ navigator.mediaDevices
     })
     .catch((err) => {});
 
+// input value
+let text = $("input");
+
```

Comment:
Why jQuery? Aren't we supposed to use React for the frontend?

Reply:
This is for a video chat application, I am using an Ejs template for view purpose to display video